crates/crypto/src/hash/mod.rs

```diff
@@ -80,7 +80,7 @@ where
     Hash::of((0b1, lhs, rhs))
 }
 
-pub trait SupportedDigest: Digest + private::Sealed + Sized + 'static {
+pub trait SupportedDigest: Digest + private::Sealed + Default + Sized + 'static {
```

Comment:
`Sha256` already implements `Default`; this just makes `#[derive(Default)]` work more places.